client/lib/accept/dialog.jsx

```diff
@@ -1,26 +1,21 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React, { PropTypes } from 'react';
 
 /**
  * Internal dependencies
  */
-var Dialog = require( 'components/dialog' );
-
-module.exports = React.createClass( {
-	displayName: 'AcceptDialog',
+import localize from 'lib/mixins/i18n/localize';
+import Dialog from 'components/dialog';
 
+const AcceptDialog = React.createClass( {
 	propTypes: {
-		// message can either be a string, an element or an array of string/elements (returned by `this.translate`)
-		message: React.PropTypes.oneOfType( [
-			React.PropTypes.string,
-			React.PropTypes.element,
-			React.PropTypes.array,
-		] ).isRequired,
-		onClose: React.PropTypes.func.isRequired,
-		confirmButtonText: React.PropTypes.string,
-		cancelButtonText: React.PropTypes.string,
+		translate: PropTypes.func,
+		message: PropTypes.node,
```

Comment:
>I didn't know there is node type :P Great to know :+1: 

As the original author of this component, neither did I at the time apparently :laughing: